src/Symfony/Component/Console/Tests/ColorTest.php

```diff
@@ -21,11 +21,11 @@ public function testAnsiColors()
         $color = new Color();
         $this->assertSame(' ', $color->apply(' '));
 
-        $color = new Color('red', 'yellow');
-        $this->assertSame("\033[31;43m \033[39;49m", $color->apply(' '));
```

Comment:
can't we keep the existing assertions instead of replacing them?

Reply:
@nicolas-grekas How do you mean? Like add new assertions instead? Seems unnecessary to me, but I can do that.

Reply:
I mean adding new assertions yes.
This would help reviews as changing an existing test case is always suspicious.